toc.json

```diff
@@ -123,7 +123,7 @@
     "key": "simcore/services/dynamic/jupyter-scipy-notebook",
     "name": "jupyter-scipy-notebook",
     "type": "dynamic",
-    "version": "2.13.0"
+    "version": "2.13.1"
```

Comment:
coupld you please check that you do not have a variant of this service in your local repo?

Reply:
yes it's strange locally it had a new version - corrected now